integration/apps/build_func.sh

```diff
@@ -66,7 +96,11 @@ setup_source_git() {
     git init
```

Comment:
Do initialization conditionally to avoid confusing stdout:

if [ ! -e .git ]; then
...